lib/Parser/Parse.cpp

```diff
@@ -7558,6 +7558,10 @@ void Parser::TransformAsyncFncDeclAST(ParseNodePtr *pnodeBody, bool fLambda)
     {
         GetCurrentFunctionNode()->sxFnc.SetUsesArguments();
     }
+    if (pnodeFncGenerator->sxFnc.CallsEval() || pnodeFncGenerator->sxFnc.ChildCallsEval())
```

Comment:
isn't this pnodeFncGenerator the dummy wrapper around the another function inside?